lerna-package-lock.json

```diff
@@ -50932,9 +50931,9 @@
 			"integrity": "sha512-ravE6m9Atw9Z/jjttRUZ+clIXogdghyZAuWJ3qEzjT+jI/dL1ifAqhZeC5VHzQp1MSt1+jxKkFNemj/iO7tVUA=="
 		},
 		"prettier": {
-			"version": "2.6.2",
-			"resolved": "https://registry.npmjs.org/prettier/-/prettier-2.6.2.tgz",
-			"integrity": "sha512-PkUpF+qoXTqhOeWL9fu7As8LXsIUZ1WYaJiY/a7McAQzxjk82OF0tibkFXVCDImZtWxbvojFjerkiLb0/q8mew=="
+			"version": "2.3.2",
```

Comment:
Why is prettier downgraded?

Reply:
Looks like fe6d4c2985ff23b8f7ecfc532abe2728501e79dc made the current prettier version no longer allowed, and the package lock was not updated.